src/run_slam.cc

```diff
@@ -82,8 +82,10 @@ void tracking(const std::shared_ptr<openvslam::config>& cfg, const std::string&
 #endif
 
     rclcpp::Rate pub_rate(10);
+    ros->setParams();
     while (rclcpp::ok()) {
-        ros->publish_pose();
+        if(!ros->camera_link_.empty())
```

Comment:
I think we need a bool variable to indicate when mapping really starts, otherwise, we'll be publishing a pose that does not exist yet. The same for the TF (which leads to errors and lots of warnings while data is not coming). Does this variable is already available at Openvslam? If it is not, where would be the best way to check this info?